src/library.js

```diff
@@ -1593,6 +1646,13 @@ const createPilingJs = (rootElement, initOptions = {}) => {
     // Consequential updates that cause new actions to be dispatched
     if (stateUpdates.has('grid')) {
       updateGrid();
+    } else if (stateUpdates.has('arrange')) {
+      changedPiles.forEach(id => {
+        const pile = pileInstances.get(id);
+        if (pile) {
```

Comment:
This can be written as follows to be more declarative:

```changedPiles.filter(id => pileInstances.has(id)).forEach(id => arrangePile(pileInstances.get(id)));```

Technically both are the same but the current code is a bit too imperative.